spec/unit/classes/master/config_spec.rb

```diff
@@ -1,20 +1,21 @@
 require 'spec_helper'
 
 describe 'puppetdb::master::config', :type => :class do
+  basefacts = {
+    :fqdn => 'puppetdb.example.com',
+    :osfamily => 'Debian',
+    :operatingsystem => 'Debian',
+    :operatingsystemrelease => '6.0',
+    :kernel => 'Linux',
+    :lsbdistid => 'Debian',
+    :lsbdistcodename => 'foo',
+    :concat_basedir => '/var/lib/puppet/concat',
+    :id => 'root',
+    :path => '/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin',
+  }
 
   let(:facts) do
-    {
-      :fqdn => 'puppetdb.example.com',
-      :osfamily => 'Debian',
-      :operatingsystem => 'Debian',
-      :operatingsystemrelease => '6.0',
-      :kernel => 'Linux',
-      :lsbdistid => 'Debian',
-      :lsbdistcodename => 'foo',
-      :concat_basedir => '/var/lib/puppet/concat',
-      :id => 'root',
-      :path => '/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin',
-    }
+    basefacts
```

Comment:
It's what we do elsewhere so we have the ability to modify basefacts in the tests, I decided to not actually modify basefacts in this test but it's still "best practice"